### PR TITLE
Issue #57 Remove pull secret from k8s secret resource

### PR DESCRIPTION
**Pull request by praveenkumar**
We have pull secret part of etcd db which is replaced by an empty
json string `{}` and also remove it from the VM disk side.

### CODE REVIEW

File: createdisk.sh [hunk 1]
```diff
@@ -166,6 +167,11 @@ if [ -z $random_string ]; then
 fi
 VM_PREFIX=${CRC_VM_NAME}-${random_string}
 
+# Replace pull secret to a null json string '{}'
```
Comment (cfergeau) [inline createdisk.sh]: 'replace ... with a null json string...'

File: createdisk.sh [hunk 2]
```diff
@@ -166,6 +167,11 @@ if [ -z $random_string ]; then
...
+${OC} --config $1/auth/kubeconfig replace -f pull-secret.yaml
+# Remove pull secret from the VM
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- rm -f /var/lib/kubelet/config.json
```
Comment (cfergeau) [inline createdisk.sh]: This should probably go after the `systemctl stop kubelet` call (and after making sure it really is stopped)? I'm wondering if it's possible that something recreates that file before it was removed. Maybe we should remove it with guestfish once the VM is shutdown?
What about https://github.com/cri-o/cri-o/pull/2510/files which apparently will duplicate the pull secret in a 3rd place? How do we ensure that from now on, the pull secret won't be duplicated in more places?

Comment (praveenkumar) [reply createdisk.sh]: @cfergeau https://github.com/cri-o/cri-o/pull/2510/files in this PR it doesn't duplicate the pull secret in 3rd place but use the existing one.  Yes, we need to keep the on disk removal part after the kublet service is stopped and all the containers/pods are deleted

Comment (cfergeau) [reply createdisk.sh]: Yup, sorry, I didn't look close enough :-/

File: createdisk.sh (hunk 1 above)
Comment (praveenkumar) [reply createdisk.sh]: Done.

**Comment by gbraad** [comment]
OK to merge or wait for the CRC side?

**Pull request merged**